1-Transformation-Scripts/cds_transformation_functions.py

```diff
@@ -180,6 +181,9 @@ def id_validation(df_dict, config, data_file, cds_log):
             for parent_column in config['PARENT_MAPPING_COLUMNS']:
                 if parent_column['node'] == node:
                     parent_id_field = parent_column['parent_node'] + '.' + parent_column['property']
+                    relationship = parent_column["relationship"]
+                    if model["Relationships"][relationship]["Mul"] == "many_to_many":
+                        mul = "many_to_many"
```

Comment:
This "many_to_many" string is used in multiple places, suggest you define a constant for it.